src/Pdk/Service/WcFrontendRenderService.php

```diff
@@ -52,7 +52,7 @@ public function renderProductSettings(PdkProduct $product): string
 
         return sprintf(
             '<div id="%s" class="panel woocommerce_options_panel">%s</div>',
-            "{$appInfo->name}_product_data",
+            "{$appInfo->name}_product_data_{$product->externalIdentifier}",
```

Comment:
Deze string wordt nu op 2 verschillende plekken gebouwd en moet wel matchen, hiervoor kun dan beter een property toevoegen aan de pdk config. Het moet wel een functie zijn dus het ziet er gek uit, maar dan houden we wel alles netjes op 1 plek.

Bijvoorbeeld:

```php
'createProductDataIdentifier' => factory(static function (): callable {
        return static function (string $productId): string {
            $appInfo = PdkFacade::getAppInfo();

            return sprintf('%s_product_data_%s', $appInfo->name, $productId);
        };
    }),
```

En dan
```php
Pdk::get('createProductDataIdentifier')($productId)
```